app/src/main/java/com/github/houseorganizer/houseorganizer/calendar/Calendar.java

```diff
@@ -211,19 +200,14 @@ public static boolean putEventStringsInData(Map<String, String> event, Map<Strin
             return false;
         }
 
-        @NonNull
-        @Override
-        public String toString() {
-            return this.title + " at " + start.toString() + ", lasts " + duration + " seconds. : " + description;
-        }
-
         @Override
         public boolean equals(Object oEvent) {
```

Comment:
Small question, did you also override hashCode()?

Reply:
oh i forgot to override it along with equals, ill do it next commit

Reply:
Alright 👍